Makefile

```diff
@@ -1,8 +1,19 @@
-PLUGIN_VERSION=2.0.0
-PLUGIN_ID=aks-clusters
+plugin_id=`cat plugin.json | python -c "import sys, json; print(str(json.load(sys.stdin)['id']).replace('/',''))"`
+plugin_version=`cat plugin.json | python -c "import sys, json; print(str(json.load(sys.stdin)['version']).replace('/',''))"`
+archive_file_name="dss-plugin-${plugin_id}-${plugin_version}.zip"
+remote_url=`git config --get remote.origin.url`
+last_commit_id=`git rev-parse HEAD`
 
 plugin:
-	cat plugin.json|json_pp > /dev/null
-	rm -rf dist
-	mkdir dist
-	zip --exclude "*.pyc" -r dist/dss-plugin-${PLUGIN_ID}-${PLUGIN_VERSION}.zip plugin.json code-env parameter-sets python-clusters python-lib python-runnables
+	@echo "[START] Archiving plugin to dist/ folder..."
+	@cat plugin.json | json_pp > /dev/null
+	@rm -rf dist
+	@mkdir dist
+	@echo "{\"remote_url\":\"${remote_url}\",\"last_commit_id\":\"${last_commit_id}\"}" > release_info.json
+	@git archive -v -9 --format zip -o dist/${archive_file_name} HEAD
```

Comment:
```suggestion
	@git archive -v -9 --add-file release_info.json --format zip -o dist/${archive_file_name} HEAD
```

Reply:
Yep, I agree about state.json, I can remove that. Makefile and .gitignore I don't have too many opinions about, maybe it could help someone else who installs the plugin, then wants to convert it to a dev plugin to make some modifications?

Reply:
Yeah that makes sense. I'm fine with just removing the state.json file

Reply:
I'm not sure I'll add the --add-file option since it's a "relatively" new option to `git archive` (for example, my `git` is too old and doesn't have it, so I might not be the only one?)

Reply:
Good point, it was added 5 versions ago but indeed git now upgrade way faster than before so it was just a year ago. If you don't have it then yeah it's probably not yet in the default everywhere so it's probably not a good idea to use it yet.